spec/simple_scheduler/scheduler_job_spec.rb

```diff
@@ -25,6 +25,16 @@
     end
   end
 
+  describe "loading a YML file with ERB tags" do
+    it 'parses the file and queues the jobs' do
+      travel_to(now) do
+        expect do
+          described_class.perform_now("spec/simple_Scheduler/config/erb_test.yml")
```

Comment:
Not sure, but it might be a typo here that's working for you locally, but not on travis because of case sensitivity for the file name?